Tables/dbo.City.sql

```diff
@@ -0,0 +1,12 @@
+
+CREATE TABLE [dbo].[City](
```

Comment:
We probably need to call this table Cities, because if you look at the rest of our table names they're all plural.  No reason this one shouldn't be plural.  It should follow the convention that the rest of our tables abide by.